src/viper/harness/config/confd.yml

```
@@ -2,20 +2,24 @@
 confd('harness:/'):
   - { src: docker/image/node/Dockerfile }
   - { src: docker/image/node/root/etc/supervisor/conf.d/client.conf }
-  - { src: docker/image/node/root/lib/task/build.sh }
   - { src: docker/image/node/root/lib/task/build/gateway.sh }
   - { src: docker/image/node/root/lib/task/build/client.sh }
```

```suggestion
  - { src: docker/image/node/root/lib/task/build/client.sh }
  - { src: docker/image/lighthouse/Dockerfile }
  - { src: docker/image/lighthouse/root/app/run.sh }
```
Do we need any of these?

---

I didn't want to add the lighthouse stuff until I was able to run the test Viper harness alone